src/globus_cli/commands/task/list.py

```diff
@@ -166,6 +166,8 @@ def task_list(
 
     def _process_filterval(prefix, value, default=None):
         if value:
+            if isinstance(value, list) and not any(value):
+                return default or ""
```

Comment:
This would also work:
```suggestion
        if value and (not isinstance(value, list) or any(value)):
```
but (of course) I think this version is moderately more readable.